dev/WebView2/WebView2.cpp

```diff
@@ -1255,14 +1255,25 @@ void WebView2::FireCoreWebView2Initialized(winrt::hresult exception)
     m_coreWebView2InitializedEventSource(*this, *eventArgs);
 }
 
-void WebView2::HandleGotFocus(const winrt::Windows::Foundation::IInspectable&, const winrt::RoutedEventArgs&) noexcept
+void WebView2::HandleGotFocus(const winrt::Windows::Foundation::IInspectable&, const winrt::RoutedEventArgs&)
 {
     if (m_coreWebView && m_xamlFocusChangeInfo.m_isPending)
     {
         // In current CoreWebView2 API, MoveFocus is not expected to fail, so set m_webHasFocus here rather than
```

Comment:
Is this comment now incorrect? Should the "m_webHasFocus = true" move after MoveFocus in the try block?

Reply:
Yep, comment mo longer applies - fixed (also forgot to push this). 

RE m_webHasFocus=true, good suggestion, thanks :-) made this change also.